Mahashree.rajendran/getting started synthetics (#9885)
DataDog/documentation: content/en/getting_started/synthetics/api_test.md

## Create an API test

> [API tests][1] help you **monitor your API endpoints and alert** you when they are failing or too slow. These checks verify that your applications are responding to requests and meet any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**. Use the [Datadog API][2] to see the full list.

Reviewer: ```suggestion
[API tests][1] help you **monitor your API endpoints and alert** you when they are failing or become too slow. These tests verify that your applications are responding to requests and meet any conditions you define, such as **response time**, **HTTP status code**, and **header or body contents**.
```
And probably remove link at the end!

    - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, which is a test web application.
    - Select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.

>     - You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.

Reviewer: ```suggestion
    - You can set tags such as `env:prod` and `app:shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
```

>     - Choose from the **managed locations** to run your test from.

Reviewer: ```suggestion
    - Choose from the **Managed Locations** to run your test from.
```

    - Click the **Test URL** button.

> {{< img src="getting_started/synthetics/api-test-config.png" alt="API test configuration"  style="width:60%;">}}

Reviewer: Ideally we would change that screenshot to reflect the env tag that is now first class citizen on the creation form and the 15 locations we now have. And I think we could set the width to 100% for all screenshots/videos.
